runner/runner.go

```diff
@@ -11,23 +11,23 @@ import (
 )
 
 type (
-	ConfigBody    = config.Body
-	ConfigGlobal  = config.Global
-	ConfigRequest = config.Request
-	ConfigRunner  = config.Runner
-	ConfigOutput  = config.Output
+	Config         = config.Global
+	RequestConfig  = config.Request
+	RequestBody    = config.RequestBody
+	RecorderConfig = config.Runner
```

Comment:
Seems strange: we ask for a "runner" field in config (json for a client) but transform it to "recorder" once we're inside the domain.

Can we improve this ?

Reply:
Good question, probably depends on the answer to this question: should we leak to the external config the fact that the runner internally uses a thing named `recorder`?

Reply:
No to the first, A to the second!
I'm ok with it then :)